Misc/NEWS.d/next/Core and Builtins/2018-11-08-15-00-58.bpo-35193.HzPS6R.rst

```diff
@@ -0,0 +1,5 @@
+Fix an off by one error in the bytecode peephole optimizer where it could read
+bytes beyond the end of bounds of a bytecode array when removing unreachable
+code after a return.  This did not always crash an interpreter, but could.
+An interpreter compiled using the clang memory sanitizer always failed.
```

Comment:
This seems a little longer and more detailed than most news entries... not sure if that's good or bad.